packages/global-cli/README.md

```diff
@@ -1,126 +1,27 @@
-## Running CLI with local modifications
+## React Native Global CLI
 
-React Native is distributed as two npm packages, `react-native-cli` and `react-native`. The first one is a lightweight package that should be installed globally (`npm install -g react-native-cli`), while the second one contains the actual React Native framework code and is installed locally into your project when you run `react-native init`.
+`react-native-cli` is a command line tool for initializing React Native projects.
 
-Because `react-native init` calls `npm install react-native`, simply linking your local github clone into npm is not enough to test local changes.
+## Usage
 
-### Introducing Sinopia
+We recommend using `react-native-cli` with `npx` so you don't need to pollute your global packages scope.
 
-[Sinopia] is an npm registry that runs on your local machine and allows you to publish packages to it. Everything else is proxied from `npmjs.com`. We'll set up sinopia for React Native CLI development. First, install it with:
+To create a new React Native Project called "AwesomeProject" you can run:
 
-    $ npm install -g sinopia
+```sh
+npx react-native-cli init AwesomeProject
+```
 
-Now you can run sinopia by simply doing:
+It will install `react-native`, `react`, `jest` and a bunch of other necessary packages from the default template.
 
-    $ sinopia
+After it's finished, your AwesomeProject should be ready to use. From this point, you should use your local `react-native` binary to run the proper [React Native CLI](../cli):
 
-Running it for the first time creates a default config file. Open `~/.config/sinopia/config.yaml` and configure it like this (note the `max_body_size`):
+```
+yarn react-native link native-dep
+```
 
-    storage: ./storage
+## Future work
 
-    auth:
-      htpasswd:
-        file: ./htpasswd
+We understand this is counter-intuitive to have two packages for interacting with React Native and it makes the first experience with the framework a bit confusing.
```

Comment:
Oh wow I didn’t realize that we need both `react-native-cli` and `yarn react-native` 😰

Should we also mention somewhere that folsk should uninstall the previous version of the CLI? Maybe at the start where you mention how to install?

Reply:
Haha yea, it's pretty bad. This is not necessary now that we advise to use global package. If someone is familiar with `npx` they can use it on their own anyway :)